enviroment_vars.conf

```
@@ -11,3 +11,4 @@ FIRST_AUDIO_PER_LANG_ONLY="true"
 AUDIO_LANGUAGES=[ "eng" ]
 SUBTITLE_LANGUAGES=[ "eng" ]
 PYTHONUNBUFFERED=1
+HIGHEST_CHANNELS="false"
```

```suggestion
HIGHEST_CHANNELS="false"

```
Should end with new line